api-gateway-config/scripts/lua/routing.lua

```diff
@@ -59,10 +59,15 @@ function _M.processCall()
       end
       -- Parse backend url
       local u = url.parse(opFields.backendUrl)
+      -- add http:// if no protocol is specified
+      if u.scheme == nil then
+        u = url.parse(utils.concatStrings({'http://', opFields.backendUrl}))
+      end
       ngx.req.set_uri(getUriPath(u.path))
       ngx.var.backendUrl = opFields.backendUrl
```

Comment:
@alexsong93 It looks that we're still storing the backendUrl here without the added scheme. I'm not sure that's a problem though as the only other place I saw `ngx.var.backendUrl` used was in mapping. Is there anywhere else we're using this variable where we would want/need the scheme attached?